modules/javafx.web/src/test/java/test/com/sun/webkit/SharedBufferTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You removed the starting copyright year, which is not OK. I recommend to revert all of the copyright changes. Then, if you prefer, you can either update it correctly to leave the starting year, or better still, leave it alone and let the script that @arapte runs periodically take care of it.

Reply:
reverted copyright date change